general/Jupyterhub/jupyter/gpu_sbatch_jupyter.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash  --login
+
+## Modify walltime and account at minimum
+#SBATCH --time=<time_request>
+#SBATCH --account=<project_handle>
+
+#SBATCH --partition=gpu-h100
```

Comment:
The reservation and partition lines can be removed. Once GPUs are generally available, these won't be necessary. Otherwise looks good!